### PR TITLE
[INFRA-34] Migrate CI from Travis to Github Actions

### PR DESCRIPTION
**Pull request by autophagy**
@knaufk I've setup the CI to run against a range of Python versions, just in case - I don't know if theres any particular version range we support, but I thought it'd be nice in case theres a version specific regression.

I've also changed the Travis specific integration test skip flag into a more general one, which got me thinking - maybe a nice thing to do on the back of this change would be to create our own Github Action, say `ververica/setup-vvp`, with a syntax of:

```
- name: Setup VVP
  uses: ververica/setup-vvp@master
  with:
    version: 2.3
    port: 8080
```

Which would spin up a small k8s deployment (minikube, or something along those lines), download + deploy a VVP community edition into it, which we (and others) could then use in our Github Action based CI pipelines. What do you think?

### CODE REVIEW

**Comment by knaufk** [comment]
Not sure I am the best person to answers these questions: 
* `jupyter-vvp` falls into the SQL Team now: @twalthr
* The custom Github Action sounds like a very nice idea. Do we also need this for other repositories?

**Comment by twalthr** [comment]
That is a nice idea @autophagy. +1 to this.

File: DEVELOP.md [hunk 1]
```diff
@@ -46,7 +46,20 @@ In order to test the upload the TestPyPi index can be used, the command to uploa
 Unit tests should be used to test functionality whereever possible.
 If integration with Ververica Platform needs to be tested, integration tests should be used.
 One of the integration tests executes example notebooks, so those should also be updated when any changes are made.
-The integration tests are currently not executed by travis, and only run locally.
+The integration tests are currently not executed by by the CI, and only run locally.
```
Comment (twalthr) [inline DEVELOP.md:49]: nit: `by by`

**Review by twalthr — APPROVED** [review]
I added one minor comment.

File: DEVELOP.md (hunk 1 above)
Comment (autophagy) [reply DEVELOP.md:49]: Ah! Good catch, thanks!

**Pull request merged**